src/SWP/Bundle/WebRendererBundle/Theme/Configuration/TenantableConfigurationProvider.php

```diff
@@ -63,14 +63,19 @@ public function __construct(
      */
     public function getConfigurations()
     {
-        $configs = array_map(
-            [$this->loader, 'load'],
-            $this->fileLocator->locateFilesNamed($this->configurationFilename)
-        );
+        // Handle there being no theme.json file
+        try {
```

Comment:
I would simply do try catch here ThemeBundle Exception (which is default \InvalidArgumentException) and throw our exception (`ThemeNotFoundException`) instead of ThemeBundle's one. If there are no themes in themes directory it should throw our exception telling that Theme cant be found.

In the `TenantAwareThemeContext` we should throw ThemeNotFoundException (as it is now) and the listener will automatically catch it and display message to install default theme, because it is available at this time already in themes folder and user should see a message just to install it